### ISSUE TITLE
Back button disable when at /home route

### ISSUE THREAD

**Issue by berthojoris** [KenAragorn/create_flutter_provider_app#2]
Why the app can't be exit when back button pressed when I at `/home` route? Why must force to close app? I see you used `WillPopScope` false

**Comment by KenAragorn** [comment]
Hi @berthojoris , yes you are right. I plan to fix it in the next few Sprint. 
Now I'm in the midway of adding another core feature which most likely needed - i18n or Internationalization feature. 

Also, thanks a lot for the comment and feedback. :)

**Comment by berthojoris** [comment]
I'm using your boilerplate project for my next project. Nice for starter and I added some small feature in register page. I added Profile name

**Comment by KenAragorn** [comment]
Hi @berthojoris ,

Good to hear that, the main reason I created this is to ease the burden and remove repetitive task especially when creating and starting a new Flutter mobile app project utilizing Firebase.

I have added additional feature that allow user to change language. Please refer to the updated README.md.

Multi-language is another feature commonly useful as we published the app to more than 1 country.

**Comment by berthojoris** [comment]
Maybe for next release you can add menu for update profile. Upload image (pick from local storage or capture from camera) and upload to Firebase storage. Integrate with push notification. I see from your another repo about push notif

**Comment by KenAragorn** [comment]
Ah, can. Will consider that upload image part for Firebase. Done a couple of times for client project. Will put that into consideration on next few Sprint.

As for Push Notification, it is better to know the basic end to end which I have a specific git project and article explaining details of it. The main reason I dont plan to combined both this project and the push notification is because, push notification itself is quite a big topic itself. Is best to learn it without the involvement of other 3rd party like state mgmt or boilerplate. 

However, most readers been very positive in that push notification guide and details step by step that I shared in the most clean and small project that covered all the topic needed in push notification. 
 
Check it here:
[shorturl.at/pwOS6](shorturl.at/pwOS6)

**Comment by berthojoris** [comment]
Yes... I already see it. Nice. Btw how about push notif triggered by another apps or via web? Via firebase console is good, but in real life app send with trigger from another source. Hope you want to create this article :)

**Comment by KenAragorn** [comment]
Yes. For now we have the initial version release - the current version. I plan to create a dashboard at a much more later stage using Flutter Web for the dashboard.

I have done a simple portfolio web using Flutter Web [(https://kenaragorn.github.io/#/),](https://kenaragorn.github.io/#/) and to be frank, I will wait for another major release first by Flutter team before I can put this into pipeline.

For now, at this for the next few months, will working more on this project on app first. :)

But, yes, thanks for the suggestion. :)
